Cargo.toml

```diff
@@ -4,9 +4,20 @@ version = "0.0.0"
 edition = "2018"
 
 [dependencies]
+anyhow = "1.0"
+git2 = "0.13.20"
 colored = { version = "1.9" }
 atty = { version = "0.2" }
 log = { version = "0.4", features = ["std"] }
+dialoguer = "0.9.0"
+indicatif = "0.16.2"
+librad = { path = "../radicle-link/librad" }
+rad-clib = { path = "../radicle-link/rad-clib" }
+rad-profile = { path = "../radicle-link/rad-profile" }
+rad-identities = { path = "../radicle-link/rad-identities" }
```

Comment:
Dependencies for the radicle libs for now should be specified with `version = "0"`, and a patch should be used in this file. Happy to explain more in RTC.